addons/godot-firebase/utils/aes.gd

```diff
@@ -0,0 +1,354 @@
+# Source: https://github.com/beamer159/AES-in-GDScript/blob/master/AES.gd
```

Comment:
If we're actually using this anywhere, we'll have to call randomize(), which I'm not keen on forcing the user to do. Can we just use the encryption that comes with Godot? I know it's probably not as powerful as AES, but I dunno that we need to include this whole new encryption algorithm in our repo.

Reply:
Actually, I don't think I use this anymore. I think we can safely delete this after making sure there's no reference to it.